Services/Init/classes/Provider/StartUpMetaBarProvider.php

```diff
@@ -128,7 +128,7 @@ private function isUserLoggedIn(): bool
 
     private function isUserOnLoginPage(UriInterface $uri): bool
     {
-        return preg_match('%^.*/login.php$%', $uri->getPath()) === 1;
+        return preg_match('%^.*(/login.php|/)$%', $uri->getPath()) === 1;
```

Comment:
```suggestion
        return preg_match('%^.*(?:/login\.php|/)$%', $uri->getPath()) === 1;
```

Just a few cents:

* Add escaping for `.`
* Use a non-capture-group